examples/tpm2-integration-test/integration_test.go

```diff
@@ -0,0 +1,57 @@
+package tpm2_integration_test
```

Comment:
Go does not like to have underscores in package names: https://blog.golang.org/package-names.

Also, normally, test code has the same package name as the package that it's testing.

Reply:
I'll fix the underscores.

IIUC, go only allows a single binary per package, since testing the binaries mean testing their interaction, it seemed that having an separate testing package was required (otherwise, it leads to circular dependencies and either duplicate tests or seemly untested packages). Or I could combine all binaries into a single one with sub commands...but that seems complex.